contracts/protocol/libraries/logic/PoolExtendedLogic.sol

```diff
@@ -0,0 +1,384 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.10;
+
+import {DataTypes} from "../types/DataTypes.sol";
+import {Errors} from "../helpers/Errors.sol";
+import {SupplyLogic} from "./SupplyLogic.sol";
+import {BorrowLogic} from "./BorrowLogic.sol";
+import {IERC20} from "../../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {INToken} from "../../../interfaces/INToken.sol";
+import {IAuctionableERC721} from "../../../interfaces/IAuctionableERC721.sol";
+import {UserConfiguration} from "../configuration/UserConfiguration.sol";
+import {Math} from "../../../dependencies/openzeppelin/contracts/Math.sol";
+import {Helpers} from "../helpers/Helpers.sol";
+import {IPoolAddressesProvider} from "../../../interfaces/IPoolAddressesProvider.sol";
+import {ValidationLogic} from "./ValidationLogic.sol";
+import {IWETH} from "../../../misc/interfaces/IWETH.sol";
+import {IPToken} from "../../../interfaces/IPToken.sol";
+import {IERC721} from "../../../dependencies/openzeppelin/contracts/IERC721.sol";
+import {PercentageMath} from "../../../protocol/libraries/math/PercentageMath.sol";
+
+library PoolExtendedLogic {
+    using Math for uint256;
+    using UserConfiguration for DataTypes.UserConfigurationMap;
+    using PercentageMath for uint256;
+
+    event ReserveUsedAsCollateralDisabled(
+        address indexed reserve,
+        address indexed user
+    );
```

Comment:
This event has defined in IPool

Reply:
Logic file can't access event in interface file.